lib/commands/simple-create-release.js

```diff
@@ -5,23 +5,34 @@ const getSelectedPackages = require('./get-selected-packages')
 const logger = require('../logger')
 const octopusApi = require('../octopus-deploy')
 
-const simpleCreateRelease = (params) => {
```

Comment:
Thanks for catching that. I must have missed my usual linting rules to watch for perens on single param arrow functions.